custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -60,6 +67,116 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun isTimeSpentFormAppeared(): Boolean {
+        try {
+            driver
+                .wait(
+                    ExpectedConditions.presenceOfElementLocated(By.id("log-work-time-logged"))
+                )
+        } catch (e: Exception) {
+            return false
+        }
+        return true
+    }
+
+    override fun cancelTimeSpentForm(): AbstractIssuePage {
+        driver
+            .wait(
+                timeout = Duration.ofSeconds(5),
+                condition = ExpectedConditions.presenceOfElementLocated(By.id("issue-workflow-transition-cancel"))
```

Comment:
Please extract and reuse locator
val transitionCancelLocator = By.id("issue-workflow-transition-cancel")